PYME/Acquire/SpoolController.py

```diff
@@ -414,7 +414,8 @@ def StartSpooling(self, fn=None, stack=None, compLevel=None,
         if stack:
             protocol = protocol_z
             protocol.dwellTime = z_dwell
-            #print(protocol)
+            stack_settings = stack if hasattr(stack, 'keys') else None
```

Comment:
Let me rest on it for a bit, but not 100% comfortable with the re-purposing of the boolean  `stack` parameter. Can see arguments both ways ...